### PR TITLE
Fix typos

### LINKED ISSUE #459: find and fix typos - GCI'19
For beginner level, to introduce to how issues are raised and PR are created.
What can be a typo : 
- any spelling issue. For example, ommit(typo) - omit(fixed typo)
- any grammatical mistake. For example, this-these, on-in etc.
Reach out to @Parikshit-Hooda to see is something relates to this issue.

### PR DESCRIPTION
**Pull request by junron**
Fixes #459 

#### Describe the changes you have made in this pr -
Fix typos in 17 files, including code, tests and documentation

[GCI task](https://codein.withgoogle.com/dashboard/task-instances/6304725222293504/)

### CODE REVIEW

**Comment by sakshi1499** [comment]
Great work on correcting the spelling errors in the comments @junron . Though I think you should not add any space between the function name and parenthesis. It might pose some problems in future.  Here is an article discussing the same issue.
https://stackoverflow.com/questions/9765942/space-after-function-name-is-wrong

**Comment by junron** [comment]
Thanks for feedback, fixed

**Comment by junron** [comment]
@sakshi1499 could you have a look?

**Comment by sakshi1499** [comment]
I am not sure why you renamed propagation to spread @junron . Rest I think everything seems good. @tachyons @ankitkataria please look into the changes and tell if something is wrong.

**Comment by junron** [comment]
@sakshi1499 I have reverted that change in the latest version.

**Comment by sakshi1499** [comment]
There is just one problem @junron all the issues are unrelated. So make different PRs for it.

**Comment by junron** [comment]
@sakshi1499 so I should have one PR for comments changes and another for code changes?

**Comment by sakshi1499** [comment]
@junron Yes

File: public/js/logix.js [hunk 1]
```diff
@@ -1454,14 +1453,13 @@ CircuitElement.prototype.isVerilogResolvable = function() {
...
-            if (this.nodeList[i].value !== undefined) {
-                this.nodeList[i].value = undefined;
-                simulationArea.simulationQueue.add(this.nodeList[i]);
-            }
-        }
+        var node = this.nodeList[i];
+        if (node.type !== NODE_OUTPUT) continue;
+        if (node.value !== undefined) continue;
+        node.value = undefined;
+        simulationArea.simulationQueue.add(node);
```
Comment (tachyons) [inline public/js/logix.js]: Just remove code refactoring from the PR, rest looks fine

Comment (sakshi1499) [reply public/js/logix.js]: We still want different PRs for code and comments right? @tachyons

Comment (junron) [reply public/js/logix.js]: Fixed @tachyons

Comment (tachyons) [reply public/js/logix.js]: It is fine as long as it is just fixing typos. Should not contain any changes in logic or code, also when fixing typo in a method name, all the instances of that method call is updated

Comment (junron) [reply public/js/logix.js]: Yep, I've reverted my changes to the code. I suppose I'll address it in a separate PR. As to the method names, I have done a manual check and changed all instances of the `removePropagation` method. In addition, all commits have been squashed

**Comment by junron** [comment]
Could @tachyons and @sakshi1499 have a look again? sorry for all the changes

**Comment by sakshi1499** [comment]
I have reviewed it too. Great work! I'll approve your task right away 💯

**Comment by junron** [comment]
Refactoring has been moved to #672

**Pull request merged**